<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwidget</class>
 <widget class="QWidget" name="mainwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Leo_xiangyi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rotate.qrc">
    <normaloff>:/source/logo2.ico</normaloff>:/source/logo2.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="SrcLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>211</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_pic">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>相似度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="informationLabel">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="responses_Text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>257</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>response information:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>42</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="IpEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>121</width>
     <height>10</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>18</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>4</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameEdit">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>3</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passwd：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswdEdit">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>27</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>4</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CommandLinkButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>18</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>243</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="rotateButton">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>79</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>109</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192.168.0.106:8081</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192.168.0.104:8081</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192.168.0.103:8081</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192.168.0.107:8081</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>33</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Connection...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RoiLabel">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>123</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ff0000;&quot;&gt;VS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>211</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="openLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="clearLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>216</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="quitLabel">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>216</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadSrcButton">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>182</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>OPEN IMAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>182</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>START COMPARE</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>SrcLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>QuitButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>responses_Text</zorder>
  <zorder>line</zorder>
  <zorder>IpEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>UsernameEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>PasswdEdit</zorder>
  <zorder>commandLinkButton</zorder>
  <zorder>radioButton</zorder>
  <zorder>line_2</zorder>
  <zorder>label_5</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_6</zorder>
  <zorder>RoiLabel</zorder>
  <zorder>label_7</zorder>
  <zorder>rotateButton</zorder>
  <zorder>ClearButton</zorder>
  <zorder>openLabel</zorder>
  <zorder>startLabel</zorder>
  <zorder>clearLabel</zorder>
  <zorder>quitLabel</zorder>
 </widget>
 <resources>
  <include location="rotate.qrc"/>
 </resources>
 <connections/>
</ui>
